int32 allied_type
float32 white_balance_auto_rate
---
int32 error